scalyr_agent/scalyr_logging.py

```diff
@@ -986,6 +997,46 @@ def filter(self, record):
             return False
 
 
+class ForceStdoutFilter(object):
+    """A filter that includes any record if it has `force_stdout` as True
+    """
+
+    def __init__(self):
+        """Initializes the filter.
+        """
+
+    def filter(self, record):
+        """Performs the filtering.
+
+        @param record: The record to filter.
+        @type record: logging.LogRecord
+
+        @return:  True if the record should be logged by this handler.
+        @rtype: bool
+        """
+        return hasattr(record, "force_stdout") and record.force_stdout
```

Comment:
Minor style thing, could also just use ``getattr`` aka ``return getattr(record, "force_stdout", False)``.

Reply:
The `hasattr` approach matches how its used in the other filters, would it be worth changing all of the uses do you think?